dcp_prototype/backend/browser/rds/browser_orm.py

```diff
@@ -0,0 +1,183 @@
+    genomic_annotation = Column(String(32))
+    genomic_annotation_biotypes = Column(String(32))
+    quantification_software = Column(String(32))
+
+
+class LibraryPrepProtocol(Base):
+    __tablename__ = "library_prep_protocol"
+
+    id = Column(Integer, primary_key=True)
+    construction_method_ontology = Column(String(16))
+    end_bias = Column(String(16))
+    nucleic_acid_source = Column(String(16))
+
+
+class Tissue(Base):
+    __tablename__ = "tissue"
+
+    id = Column(Integer, primary_key=True)
+    tissue_ontology = Column(String(16))
+
+
+class Species(Base):
+    __tablename__ = "species"
+
+    id = Column(Integer, primary_key=True)
+    species_ontology = Column(String(16))
+
+
+class DataRepository(Base):
+    __tablename__ = "data_repository"
+
+    id = Column(Integer, primary_key=True)
+    name = Column(String(32))
+
+
+class Contributor(Base):
+    __tablename__ = "contributor"
+
+    id = Column(Integer, primary_key=True)
+    name = Column(String(32))
+    institution = Column(String(32))
+    email = Column(String(32))
+
+
+class ExternalAccession(Base):
+    __tablename__ = "external_accession"
+
+    id = Column(Integer, primary_key=True)
+    project_id = Column(ForeignKey("project.id"), nullable=False)
+    data_repository_id = Column(ForeignKey("data_repository.id"), nullable=False)
+    accession = Column(String(32))
+
+    project = relationship("Project")
+    data_repository = relationship("DataRepository")
+
+
+class LibraryPrepProtocolJoinProject(Base):
+    __tablename__ = "library_prep_protocol_join_project"
+
+    id = Column(Integer, primary_key=True)
+    library_prep_protocol_id = Column(ForeignKey("library_prep_protocol.id"), nullable=False)
+    project_id = Column(ForeignKey("project.id"), nullable=False)
+
+    library_prep_protocol = relationship("LibraryPrepProtocol")
+    project = relationship("Project")
+
+
+class TissueJoinProject(Base):
+    __tablename__ = "tissue_join_project"
+
+    id = Column(Integer, primary_key=True)
+    tissue_id = Column(ForeignKey("tissue.id"), nullable=False)
+    project_id = Column(ForeignKey("project.id"), nullable=False)
+
+    tissue = relationship("Tissue")
+    project = relationship("Project")
+
+
+class SpeciesJoinProject(Base):
+    __tablename__ = "species_join_project"
+
+    id = Column(Integer, primary_key=True)
+    species_id = Column(ForeignKey("species.id"), nullable=False)
+    project_id = Column(ForeignKey("project.id"), nullable=False)
+
+    species = relationship("Species")
+    project = relationship("Project")
+
+
+class ContributorJoinProject(Base):
+    __tablename__ = "contributor_join_project"
+
+    id = Column(Integer, primary_key=True)
+    contributor_id = Column(ForeignKey("contributor.id"), nullable=False)
+    project_id = Column(ForeignKey("project.id"), nullable=False)
+
+    contributor = relationship("Contributor")
+    project = relationship("Project")
+
+
+# uncomment to drop and recreate all tables
```

Comment:
This shouldn't be here?

Reply:
Will clean up "create database" code into a separate script